x-pack/packages/ml/aiops_components/src/dual_brush/dual_brush.tsx

```diff
@@ -234,6 +234,9 @@ export function DualBrush({
           .attr('id', (b: DualBrush) => {
             return 'aiops-brush-' + b.id;
           })
+          .attr('data-test-subj', (b: DualBrush) => {
+            return 'aiopsBrush' + b.id.charAt(0).toUpperCase() + b.id.slice(1);
```

Comment:
I'm assuming there's a specific reason why we are doing `b.id.charAt(0).toUpperCase() + b.id.slice(1)` instead of just `b.id`... Can we add a comment for why?

Reply:
Added a comment here: [9f8d948](https://github.com/elastic/kibana/pull/138661/commits/9f8d948d34e50261b482e6845b6030213c24f9ae).